Set `CONDA_STORE_DIR` to `platformdirs.user_data_path` (#786)
conda-incubator/conda-store: docusaurus-docs/conda-store/references/configuration-options.md

conda-store-worker --config <path-to-conda-store-config.py>
```

### Data directory

The `CONDA_STORE_DIR` Python variable specifies the conda-store data directory,
which is used by some of the configuration options mentioned below, like
`CondaStore.store_directory` and `LocalStorage.storage_path`. This variable

> relies on the [platformdirs] library to select the recommended user data

Reviewer: ```suggestion
relies on the [`platformdirs`][platformdirs] library to select the recommended user data
```

I think this is the right syntax, isn't it?

> location on each platform:

Reviewer: ```suggestion
location on each platform. On most systems, this will default to:
```

- Linux: `/home/<USER>/.local/share/conda-store`
- Windows: `C:\Users\<USER>\AppData\Local\conda-store\conda-store`
- macOS: `/Users/<USER>/Library/Application Support/conda-store`.

> The platform user data directory prefix, which is before `conda-store` above,

Reviewer: ```suggestion
The platform user data directory prefix, which is the parent of the `conda-store` directory above,
```

should correspond to the following environment variables:

- Linux: `$XDG_DATA_HOME`
- Windows: `%LOCALAPPDATA%`
- macOS: no dedicated environment variable.

Note that whether these environment variables are actually used by

> [platformdirs] is up to the library authors and can be changed at any time.

Reviewer: ```suggestion
[`platformdirs`][platformdirs] is up to the library authors and can be changed at any time.
```
